src/app/pcasts/controllers/sign_out_controller.py

```diff
@@ -0,0 +1,18 @@
+from . import *
+
+class SignOutController(AppDevController):
+
+  def get_path(self):
+    return '/users/sign_out/'
+
+  def get_methods(self):
+    return ['POST']
+
+  @authorize
+  def content(self, **kwargs):
+    user = kwargs.get('user')
+    success = sessions_dao.deactivate_session(user.id)
+    if success:
+      return dict()
+    else:
+      raise Exception('Issue logging out!')
```

Comment:
It looks like the only time this would be thrown is if the does not currently have an active session, so maybe we could make the error message a bit more description to reflect that